README.md

```diff
@@ -39,14 +39,14 @@ This can be installed [here](https://www.rabbitmq.com/) or via the package manag
 First, boost and cmake is required to generate the libraries below, you may install these using your appropriate package manager (such as apt-get on debian).
 The following are the commands that could be used on a debian based distro to install these dependencies if you do not already have them.
 
-`sudo apt-get install cmake libboost-dev`
+`sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev`
```

Comment:
Should we add these new ones to the travis file as well? I think it would reduce any future confusion when someone compares the commands from the README and travis.